app/models/concerns/searchable.rb

```diff
@@ -20,8 +20,35 @@ def search(name_or_phone_str, lines = LINES)
       sort_and_limit_patient_matches(all_matching_names, all_matching_phones, all_matching_identifiers)
     end
 
+    def search_users(name_str)
+      name_regexp = /#{Regexp.escape(name_str)}/i
+      all_matching_names = find_name_matches_user name_regexp
+
+      sort_and_limit_user_matches(all_matching_names)
+
+    end
+
```

Comment:
Got it, i'm going to move all my helpers into concerns ```UserSearchable``` and keep original ```Searchable``` untouched. I'll try to remember to migrate into ```PatientSearchable``` after this PR is merged.